src/pages/workspace/WorkspaceNewRoomPage.tsx

```diff
@@ -79,22 +79,22 @@ function WorkspaceNewRoomPage() {
         }
         return '';
     });
-    const isPolicyAdmin = useMemo(() => {
+    const isUserPolicyAdmin = useMemo(() => {
         if (!policyID) {
             return false;
         }
 
-        return ReportUtils.isPolicyAdmin(policyID, policies);
+        return isPolicyAdmin(policyID, policies);
     }, [policyID, policies]);
     const [newRoomReportID, setNewRoomReportID] = useState<string>();
 
     /**
      * @param values - form input values passed by the Form component
      */
     const submit = (values: FormOnyxValues<typeof ONYXKEYS.FORMS.NEW_ROOM_FORM>) => {
-        const participants = [session?.accountID ?? -1];
-        const parsedDescription = ReportUtils.getParsedComment(values.reportDescription ?? '', {policyID});
-        const policyReport = ReportUtils.buildOptimisticChatReport(
+        const participants = session?.accountID ? [session.accountID] : [];
```

Comment:
```suggestion
        const participants = [session?.accountID ?? CONST.DEFAULT_NUMBER_ID];
```

NAB: we are allowed to default to number id here, but I think your way makes more sense